cmd/satellite/repairer.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
it would be nice for everything to have the same patttern. So if we put the `cmdRepairerRun` in its own file, it would be nice for the apiRun to be the same

Reply:
sounds good